[2021-05-07 15:38:28,057] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: python_dag.helloworld 2021-05-07T10:08:20.834783+00:00 [queued]>
[2021-05-07 15:38:28,436] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: python_dag.helloworld 2021-05-07T10:08:20.834783+00:00 [queued]>
[2021-05-07 15:38:28,436] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-05-07 15:38:28,436] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-05-07 15:38:28,436] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-05-07 15:38:28,492] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): helloworld> on 2021-05-07T10:08:20.834783+00:00
[2021-05-07 15:38:28,525] {standard_task_runner.py:52} INFO - Started process 11108 to run task
[2021-05-07 15:38:28,557] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'python_dag', 'helloworld', '2021-05-07T10:08:20.834783+00:00', '--job-id', '47', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/test_dag_python.py', '--cfg-path', '/tmp/tmp8c3ue_xx', '--error-file', '/tmp/tmpxn_ssp84']
[2021-05-07 15:38:28,570] {standard_task_runner.py:77} INFO - Job 47: Subtask helloworld
[2021-05-07 15:38:29,413] {logging_mixin.py:104} INFO - Running <TaskInstance: python_dag.helloworld 2021-05-07T10:08:20.834783+00:00 [running]> on host jaarora-WX-1.corp.adobe.com
[2021-05-07 15:38:30,390] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=jacrod2901@gmail.com
AIRFLOW_CTX_DAG_OWNER=jacrod2901
AIRFLOW_CTX_DAG_ID=python_dag
AIRFLOW_CTX_TASK_ID=helloworld
AIRFLOW_CTX_EXECUTION_DATE=2021-05-07T10:08:20.834783+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-05-07T10:08:20.834783+00:00
[2021-05-07 15:38:30,393] {python.py:118} INFO - Done. Returned value was: Printing hello world!
[2021-05-07 15:38:30,859] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=python_dag, task_id=helloworld, execution_date=20210507T100820, start_date=20210507T100828, end_date=20210507T100830
[2021-05-07 15:38:31,470] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-05-07 15:38:31,513] {local_task_job.py:146} INFO - Task exited with return code 0
